src/uct/ib/dc/dc_mlx5_ep.c

```diff
@@ -308,13 +354,12 @@ ucs_status_t uct_dc_mlx5_ep_am_short(uct_ep_h tl_ep, uint8_t id, uint64_t hdr,
     uct_rc_mlx5_am_hdr_fill(&cache.am_hdr.rc_hdr, id);
     cache.am_hdr.am_hdr = hdr;
 
-    UCT_DC_MLX5_IFACE_TXQP_GET(iface, ep, txqp, txwq);
```

Comment:
You can also remove `UCT_DC_MLX5_TXQP_DECL(txqp, txwq);` from the function.

Reply:
thanks, good catch
fixed